aksel.nav.no/website/app/globals.css

```diff
@@ -10,6 +10,7 @@
   --website-header-height-width-border: calc(var(--website-header-height) + 1px);
   --website-font-mono: ui-monospace, "Menlo", "Monaco", "Cascadia Mono", "Segoe UI Mono", "Roboto Mono", "Oxygen Mono",
     "Ubuntu Monospace", "Source Code Pro", "Fira Mono", "Droid Sans Mono", "Courier New", monospace;
+  --a-z-index-popover: 1000;
```

Comment:
Since this is a standalone case, i think we can just set it to 9999 directly in `.floating`
```suggestion
```